frontend/ui/elements/reader_menu_order.lua

```diff
@@ -23,7 +23,6 @@ local order = {
         "highlight_options",
         "change_font",
         "hyphenation",
-        "read_timer",
```

Comment:
Any reason to prefer this plugin to be placed in tools? IMO, it's more like a "reading experience".